src/PinterestPylintLinter.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Copyright 2020 Pinterest, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Base class for external Pylint-based linters.
+ */
+class PinterestPylintLinter extends PythonExternalLinter {
+
+  private $virtualenvs = array('.venv');
+
+  public function getPythonBinary() {
+    return 'pylint';
+  }
+
+  public function shouldExpectCommandErrors() {
+      return true;
+  }
+
+  public function getInfoName() {
+    return 'pinterest-pylint';
+  }
+
+  public function getInfoDescription() {
+    return pht("Up-to-date replacement for Arcanist's built-in pylint linter");
+  }
+
+  public function getInfoURI() {
+    return 'https://docs.pylint.org/en/';
+  }
+
+  public function getLinterName() {
+    return 'PINTEREST-PYLINT';
```

Comment:
This can just be `PYLINT`; it's what shows up in the linter output.

```suggestion
    return 'PYLINT';
```